gatsby-config.js

```diff
@@ -270,9 +269,7 @@ module.exports = {
         {
             resolve: 'gatsby-plugin-svgr',
             options: {
-                rule: {
-                    include: /svg/, // See below to configure properly
-                },
+                include: /svg/,
```

Comment:
```suggestion
                include: /svg/,
```
Is this necessary?

Reply:
@mohammad-hashemi-deriv

Reply:
This config was there since long ago to only include SVGs under `/svg/` folder.
By removing the config, the plugin will work the same but it also looks into other folders for SVG files.
You're right @yashim-deriv we can remove the config and only keep `'gatsby-plugin-svgr'` in the `plugins` array.